Golem.Tools/App/app.py

```diff
@@ -211,20 +211,29 @@ async def print_usage():
                 custom_url = "/sdapi/v1/txt2img"
                 url = activity._api.api_client.configuration.host + f"/activity/{activity.id}/proxy_http_request" + custom_url
                 payload = '{"prompt": "example prompt"}'.replace("\"", "\\\"")
```

Comment:
Code produces 
` -Body (@{ {\"prompt\": \"example prompt\"} }|ConvertTo-Json) `
but for `ps1` it should
`-Body (@{"prompt"="cat reading a book";}|ConvertTo-Json)`